docs/src/Combinatorics/graphs.md

```diff
@@ -61,6 +61,7 @@ label!
 degree(g::Graph, v::Int)
 indegree(g::Graph{Directed}, v::Int)
 outdegree(g::Graph{Directed}, v::Int)
+leaves
```

Comment:
This probably needs a signature as well, and it should probably be in the same section as `n_leaves`?